changelogs/fragments/6002-keycloak-identity-provider-mappers-deterministic-changes

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
Please add a `.yml` extension to this file :)

Reply:
Oups sorry my bad.

This is fixed.